redaxo/src/addons/structure/plugins/content/lib/article_content_base.php

```diff
@@ -439,7 +439,10 @@ public function getArticleTemplate()
 
             $TEMPLATE = new rex_template($this->template_id);
             $tplContent = $this->replaceCommonVars($TEMPLATE->getTemplate());
-            require rex_stream::factory('template/' . $this->template_id, $tplContent);
+
+            rex_timer::measure('Template: '.($TEMPLATE->getKey() ?? $TEMPLATE->getId()), function () use ($tplContent) {
```

Comment:
Hier stellt sich wieder die Frage, ob wir einheitlich die ID nehmen wollen, oder so wie jetzt, den Key, falls vorhanden. (siehe Screenshot oben)

Falls so lassen, würden wir hier auch wieder vom Key-Cache profitieren.

Reply:
Insofern wir den cache haben und es hier nutzen, sollten wir es auch in der stream url nutzen (oder annbeiden stellen nur die id)

Reply:
Die andere Stelle ist halt deutlich problematischer. Zieht sich bis ins Developer-Addon, wo ich dann Anpassungen machen müsste.
Hier hingegen ist es ja nur ein Label.
Daher dürfte es für mich auch unterschiedlich sein an den beiden Stellen.
Für mich ist aber auch ok, wenn wir dann hier auch nur die ID verwenden.

Reply:
Dann würd ich jetzt erstmal überall ID lassen, wenn keys zu große kreise zieht

Reply:
Wobei ich es eigentlich schon schade finde, da die Keys nun mal sprechender sind.
Also ich würde eigentlich doch dafür plädieren, hier die Keys zu verwenden, und dann bei den Streams ggf. später nochmal schauen, ob man da auch auf Keys wechseln kann (wobei es für mich wie gesagt auch nicht zwingend gleich sein muss, sehe keinen direkten Zusammenhang).